nix/process-compose/default.nix

```diff
@@ -50,5 +56,26 @@ in
         exec process-compose -p ${toString config.port} "$@"
       '';
     };
+  config.outputs.testPackage =
+    let
+      mergedYaml = pkgs.runCommand "${name}-merged.yaml" { buildInputs = [ pkgs.yq-go ]; } ''
+        yq '. *= load("${config.outputs.settingsYamlOverlay}")' ${config.outputs.settingsYaml} > $out
+      '';
```

Comment:
Do we really need Nix to do the merging, when process-compose already does that for us?

https://github.com/F1bonacc1/process-compose/wiki/Multiple-Compose-Files

Reply:
How do we print the merged yaml when `debug = true;` if process-compose does the merging?

Reply:
```nix
${if config.debug then "cat ${config.outputs.settingsYaml} ${config.outputs.settingsYamlOverlay}" else ""}
export PC_CONFIG_FILES="${config.outputs.settingsYaml} ${config.outputs.settingsYamlOverlay}"
```

Reply:
I was considering this but because the user will have to see manually what's overridden, I gave up on this approach.

I will try it out and see if that works.

Reply:
Honestly I feel like we should just get rid of `debug` option. The user can always inspect the script and look at the paths used in it if they want to see the YAML.

Reply:
Turns out this doesn't work when I test it:
```bash
export PC_CONFIG_FILES="${config.outputs.settingsYaml} ${config.outputs.settingsYamlOverlay}"
```
But this worked:
```bash
exec process-compose -p ${toString config.port} -f ${config.outputs.settingsYaml} -f ${config.outputs.settingsYamlOverlay} "$@"
```